workspaces/resource-optimization/plugins/resource-optimization-common/src/generated/apis/OptimizationsApi.client.ts

```diff
@@ -65,12 +65,14 @@ export class OptimizationsApiClient {
     },
     options?: RequestOptions,
   ): Promise<TypedResponse<RecommendationBoxPlots>> {
+    console.log('Request Data:', request, request.path.recommendationId);
+
```

Comment:
Did you change this file manually?

Reply:
@jkilzi As discussed I will rebase this once you push the change for handling `recommendation_id` for `recommendations/id` API endpoint.